run-build

```diff
@@ -196,45 +194,12 @@ class ImageBuildRoot(object):
         self._run(['systemd-sysusers'])
         self._run(['systemd-tmpfiles', '--create', '--remove', '--boot', '-E'])
 
-        # Move the dpkg database back to its normal location
-        os.rename(
-            os.path.join(self.builddir, 'usr/share/dpkg/database'),
-            os.path.join(self.builddir, 'var/lib/dpkg'),
-        )
-
         # update-catalog requires /var/lib/sgml-base to exist
         os.makedirs(os.path.join(self.builddir, 'var/lib/sgml-base'))
 
         # update-xmlcatalog requires /var/lib/xml-core to exist
         os.makedirs(os.path.join(self.builddir, 'var/lib/xml-core'))
 
-        log.info('Installing buildroot packages')
```

Comment:
This is separate from the package metadata. Can you split it into a different commit?

Reply:
Done in 34a89ddaf3c80f8a78c302e516241410cfa0aa0c